RCPTT_Tests/ConfigurationTests/DeletingAllBlocksDeletesAllGroups.test

```diff
@@ -0,0 +1,113 @@
+    add_block_to_group "NEW_GROUP" "NEW_BLOCK_02"
+    add_block_to_group "NEW_GROUP" "NEW_BLOCK_03"
+    
+    add_block_to_group "NEW_GROUP_1" "NEW_BLOCK_04"
+    add_block_to_group "NEW_GROUP_1" "NEW_BLOCK_05"
+    add_block_to_group "NEW_GROUP_1" "NEW_BLOCK_06"
+    
+    add_block_to_group "NEW_GROUP_2" "NEW_BLOCK_07"
+    add_block_to_group "NEW_GROUP_2" "NEW_BLOCK_08"
+    add_block_to_group "NEW_GROUP_2" "NEW_BLOCK_09"
+    
+    save_config
+}
+
+assert_wait_for_dialog_closed
+
+try -times 500 -delay 200 -command {
+	with [get-view Blocks] {
+	    // Check for existence of first group and its three blocks
+	    get-label "NEW_GROUP" | get-property caption | equals "NEW_GROUP" | verify-true
+	    get-label "NEW_BLOCK_01:" | get-property caption | equals "NEW_BLOCK_01:" | verify-true
+	    get-label "NEW_GROUP" | get-property caption | equals "NEW_GROUP" | verify-true
+	    get-label "NEW_BLOCK_02:" | get-property caption | equals "NEW_BLOCK_02:" | verify-true
+	    get-label "NEW_GROUP" | get-property caption | equals "NEW_GROUP" | verify-true
+	    get-label "NEW_BLOCK_03:" | get-property caption | equals "NEW_BLOCK_03:" | verify-true
+	    
+	    // Check for existence of second group and its three blocks
+	    get-label "NEW_GROUP_1" | get-property caption | equals "NEW_GROUP_1" | verify-true
+	    get-label "NEW_BLOCK_04:" | get-property caption | equals "NEW_BLOCK_04:" | verify-true
+	    get-label "NEW_GROUP_1" | get-property caption | equals "NEW_GROUP_1" | verify-true
+	    get-label "NEW_BLOCK_05:" | get-property caption | equals "NEW_BLOCK_05:" | verify-true
+	    get-label "NEW_GROUP_1" | get-property caption | equals "NEW_GROUP_1" | verify-true
+	    get-label "NEW_BLOCK_06:" | get-property caption | equals "NEW_BLOCK_06:" | verify-true
+	    
+	    // Check for existence of third group and its three blocks 
+	    get-label "NEW_GROUP_2" | get-property caption | equals "NEW_GROUP_2" | verify-true
+	    get-label "NEW_BLOCK_07:" | get-property caption | equals "NEW_BLOCK_07:" | verify-true
+	    get-label "NEW_GROUP_2" | get-property caption | equals "NEW_GROUP_2" | verify-true
+	    get-label "NEW_BLOCK_08:" | get-property caption | equals "NEW_BLOCK_08:" | verify-true
+	    get-label "NEW_GROUP_2" | get-property caption | equals "NEW_GROUP_2" | verify-true
+	    get-label "NEW_BLOCK_09:" | get-property caption | equals "NEW_BLOCK_09:" | verify-true
+	}
+}
+
+edit_current_config
+assert_wait_for_dialog_closed
+
+// Procedure to delete a block
+proc delete_block [val cell_number] {
+	with [get-window "Edit Configuration"] {
+	    get-tab-folder | get-tab-item Blocks | click
+	    get-table | get-cell $cell_number 0 | click
+	}
+	set-dialog-result MessageBox 32
+	with [get-window "Edit Configuration"] {
+		get-button "Delete Block" | click
+	}
+}
+
+delete_block "8"
+delete_block "7"
+delete_block "6"
+delete_block "5"
+delete_block "4"
+delete_block "3"
+delete_block "2"
+delete_block "1"
+delete_block "0"
```

Comment:
We figured out how to do loops in RCPTT. See CurrentConfigNotInDeleteConfigDialog for example. Can we do that here please